ee/clickhouse/models/session_recording_event.py

```diff
@@ -49,3 +50,18 @@ def create_session_recording_event(
         capture_exception(Exception(f"Session recording event data too large - {len(snapshot_data_json)}"))
 
     return str(uuid)
+
+
+def get_session_recording_count_for_teams_and_period(
```

Comment:
In **instance** status reports we select the length of the session recording events table, which is technically useful, but by aggregating here on `session_id` we'll report a more accurate number of sessions from the users' perspectives.